paper-menu-button.html

```diff
@@ -305,6 +305,17 @@
         },
 
         /**
+         * Toggles the drowpdown content between opened and closed.
+         */
+        toggle: function() {
```

Comment:
It was necessary because with only `open` being called, the `paper-dropdown-close` event would never be fired.